filecoin-proofs/src/constants.rs

```diff
@@ -93,6 +121,23 @@ pub type DefaultPieceDomain = <DefaultPieceHasher as Hasher>::Domain;
 pub type DefaultTreeHasher = storage_proofs::hasher::PoseidonHasher;
 pub type DefaultTreeDomain = <DefaultTreeHasher as Hasher>::Domain;
 
+pub type DefaultBinaryTree = storage_proofs::merkle::BinaryMerkleTree<DefaultTreeHasher>;
+pub type DefaultOctTree = storage_proofs::merkle::OctMerkleTree<DefaultTreeHasher>;
+pub type DefaultOctLCTree = storage_proofs::merkle::OctLCMerkleTree<DefaultTreeHasher>;
+
+pub type SectorShape2KiB = LCTree<DefaultTreeHasher, U8, U0, U0>;
+pub type SectorShape4KiB = LCTree<DefaultTreeHasher, U8, U2, U0>;
+pub type SectorShape16KiB = LCTree<DefaultTreeHasher, U8, U8, U0>;
+pub type SectorShape32KiB = LCTree<DefaultTreeHasher, U8, U8, U2>;
+pub type SectorShape8MiB = LCTree<DefaultTreeHasher, U8, U0, U0>;
+pub type SectorShape16MiB = LCTree<DefaultTreeHasher, U8, U2, U0>;
+pub type SectorShape512MiB = LCTree<DefaultTreeHasher, U8, U0, U0>;
+pub type SectorShape1GiB = LCTree<DefaultTreeHasher, U8, U2, U0>;
+pub type SectorShape32GiB = LCTree<DefaultTreeHasher, U8, U8, U0>;
+pub type SectorShape64GiB = LCTree<DefaultTreeHasher, U8, U8, U2>;
```

Comment:
Let's at least clean this up by naming the shape types and assigning those to sectors. This is a note to self. Either I will do it, or it can slide until later.